flow/ProtocolVersion.h

```diff
@@ -21,60 +21,21 @@
 #pragma once
 #include <cstdint>
 
-namespace fdb_versions {
+#define PROTOCOAL_VERSION_FEATURE(v, x)                                                                                \
```

Comment:
There's a typo in "PROTOCOAL" that'll need to be fixed here and in its usages.

Reply:
Haha - good catch! I also don't like coal 😆 I pushed a fix.